keyboards/mjolnir/v1/info.json

```diff
@@ -0,0 +1,141 @@
+{
+    "keyboard_name": "mjolnir/v1",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "nkro": true,
+        "mousekey": true,
+        "encoder": true,
+        "console": true,
+        "debug": true
+    },
+    "dynamic_keymap": {
+        "layer_count": 6
+    },
+    "encoder": {
+        "rotary": [
+          {"pin_a": "GP28", "pin_b": "GP27"}
+        ]
+    },
+    "matrix_pins": {
+        "rows": ["GP12", "GP13", "GP14", "GP15", "GP16", "GP26"],
+        "cols": ["GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP11", null, "GP15", "GP21", "GP29"]        
+    },
+    "split": {
+        "enabled": true,
+        "serial": {
+            "driver": "vendor"
+        },
+        "matrix_pins": {
+            "right": {
+                "rows": ["GP12", "GP13", "GP14", "GP15", "GP16", "GP26"],
+                "cols": ["GP11", "GP9", "GP8", "GP7", "GP6", "GP5", "GP4", "GP10", "GP15", "GP21", "GP29"]     
+            }
+        },
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP28", "pin_b": "GP27"}
+                ]
+            }
+        }
+    },
+    "layouts": {
+        "LAYOUT_split": {
```

Comment:
This layout does not look correct:

<img width="833" height="242" alt="image" src="https://github.com/user-attachments/assets/228e0d3d-8f7c-4be1-8c95-0a771c4044e2" />